src/results/proforma.jl

```diff
@@ -33,7 +33,8 @@ return Dict(
     "offtaker_discounted_annual_free_cashflows" => Float64[],
     "offtaker_discounted_annual_free_cashflows_bau" => Float64[],
     "developer_annual_free_cashflows" => Float64[],
-    "initial_capital_costs_after_incentives_without_macrs" => 0.0 # Initial capital costs after ibi, cbi, and ITC incentives
```

Comment:
Just confirming this was deemed to be not useful for users, or incorrect?

Reply:
Actually I kept it, but removed "initial_" because I'm also include the present value/cost of battery replacements. So it's mainly just a name change and made it consistent with the naming of the new capital_cost_ input that I added.